src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -560,6 +628,10 @@ export default {
 			return moment(this.timestamp * 1000).format('lll')
 		},
 
+		editedDateTime() {
+			return moment(this.messageObject.lastEditTimestamp * 1000).format('lll')
```

Comment:
_Note:_
We'll need to run through these files after one of the PR is merged to avoid conflicts

Reply:
Actual note: Who will deal with conflicts 😛 ?